workspace-manager-clienttests/src/main/resources/suites/BasicResiliency.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "BasicResiliency",
+  "description": "Test deleting all initially created pods",
```

Comment:
Nit: This is a suite, so perhaps "Example resilience tests"?

Reply:
Changed description of the suite.